unstructured_inference/models/tables.py

```diff
@@ -65,7 +65,11 @@ def initialize(
 
         try:
             logger.info("Loading the table structure model ...")
+
+            logging.set_verbosity_error()
```

Comment:
it would be nicer to:
- cache the current verbosity in a variable with [get_verbosity](https://huggingface.co/docs/transformers/en/main_classes/logging#transformers.utils.logging.get_verbosity)
- set the verbosity to the cached value with `set_verbosity`